plugins/language_sh.lua

```diff
@@ -3,7 +3,7 @@ local syntax = require "core.syntax"
 
 syntax.add {
   name = "Shell script",
-  files = { "%.sh$" },
+  files = { "%.sh$", "%.bash$", ".bashrc", ".bash_profile", ".profile" },
```

Comment:
You forgot to escape the `.` with `%,` also adding `^` at beggining to only match files that start as written and `$` end also as written:

```lua
"^%.bashrc$", "^%.bash_profile$"
```

Reply:
Oh, I'm sorry. Let me fix that real quick

Reply:
fixed it :)